lib/rules/web/admin_console/4.1/route.xyaml

```diff
@@ -93,3 +93,69 @@ upload_destination_ca_crt_file:
     visible: false
     type: filefield
     op: set <destination_ca_certificate>
+filter_and_show_accepted_routes:
+  params:
+    status_text: Rejected
+  action: disable_status_filter
+filter_and_show_rejected_routes:
+  params:
+    status_text: Accepted
+  action: disable_status_filter
+disable_status_filter:
+  element:
+    selector:
+      xpath: //a[contains(@class,'row-filter__box--active') and contains(@class,'row-filter__box') and text()='<status_text>']
+    op: click
+enable_status_filter:
```

Comment:
`enable_status_filter` is not used in this PR, let team decide to keep or discard

Reply:
do we need use two different action for disable/enable the filter?  Maybe use the same one`//a[@class='row-filter__box' and text()='<status_text>']` is enough and you also will have a check the filter results then. And does the rule can be used as common rules put in  `common_ui_elements` file?

Reply:
<img width="1147" alt="Screen Shot 2020-04-01 at 11 55 50 AM" src="https://user-images.githubusercontent.com/12692381/78097778-c95f0300-740f-11ea-837e-46b67ba4cf3e.png">
This is the definition of filters in different state

Reply:
When status is checked/enabled, it has class `class="row-filter__box row-filter__box--active`, while status is unchecked/disabled,  it has class `class="row-filter__box row-filter__box--empty`

Reply:
> And does the rule can be used as common rules put in common_ui_elements file?

Yes, it's common rules, I can move to either `common_ui_elements.xyaml` or `filters.xyaml`

Reply:
OK, got it